aten/src/ATen/test/cuda_caching_host_allocator_test.cpp

```diff
@@ -9,6 +9,109 @@
 
 constexpr int64_t N = 100;
 
+// NOTE: please leave this as the first test to ensure that
+// the allocator is not used and stats are zero.
+TEST(CachingHostAllocatorTest, check_stats) {
```

Comment:
can we have a python test instead? We generally strongly prefer python tests (as that is what users call)

Reply:
Yes, added a new test function on the Python side.